pkg/volumeclaimbinder/persistent_volume_claim_binder_test.go

```diff
@@ -0,0 +1,200 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volumeclaimbinder
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/latest"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/resource"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client/testclient"
+)
+
+func TestExampleObjects(t *testing.T) {
+
+	scenarios := map[string]struct {
```

Comment:
concretely, taking your code

```
scenarios := map[string]struct {
		expected interface{}
}{
	"claims/claim-01.yaml": &api.PersistentVolumeClaim{
		Spec: api.PersistentVolumeClaimSpec{
			AccessModes: []api.AccessModeType{api.ReadWriteOnce},
			Resources: api.ResourceRequirements{
				Requests: api.ResourceList{
					api.ResourceName(api.ResourceStorage): resource.MustParse("3Gi"),
				},
			},
		},
	},
```

The only difference is that I omitted "{\n\t\t\texpected: " and " one closing curly.